src/IssueViz.UnitTests/Editor/LocationNavigatorTests.cs

```diff
@@ -28,94 +28,79 @@
 using SonarLint.VisualStudio.Integration;
 using SonarLint.VisualStudio.Integration.UnitTests;
 using SonarLint.VisualStudio.IssueVisualization.Editor;
-using SonarLint.VisualStudio.IssueVisualization.Models;
-using SonarLint.VisualStudio.IssueVisualization.Selection;
 
-namespace SonarLint.VisualStudio.IssueVisualization.UnitTests.Selection
+namespace SonarLint.VisualStudio.IssueVisualization.UnitTests.Editor
 {
     [TestClass]
-    public class LocationChangedEventListenerTests
+    public class LocationNavigatorTests
```

Comment:
Replaced the EventListener with a simple service and modified the tests accordingly.